hardhat.config.ts

```diff
@@ -70,6 +74,10 @@ const config: HardhatUserConfig = {
             }),
             gasPrice: 0,
         },
+        'local-fork': {
+            ...bellecourBase,
+            url: 'http://127.0.0.1:8545',
+        },
```

Comment:
```suggestion
```
We already have:
`IS_LOCAL_FORK=true npx hardhat <cmd> --network hardhat`
&
`IS_LOCAL_FORK=true npx hardhat <cmd> --network external-hardhat`